mop1/project1_1.py

```diff
@@ -5,16 +5,19 @@
 
 def poli_twitter_analysis():
 	"""This function parses Twitter to determine the average sentiment towards a user-defined political figure"""
-	
```

Comment:
Sorry about the spacing changes... I'll figure out a way to avoid this in the future